src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -64,6 +67,9 @@ public ArrayList()
         // initially empty, but will have room for the given number of elements
         // before any reallocations are required.
         // 
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the specified initial capacity.</summary>
```

Comment:
I just verified the simplifier should properly reduce cref names.  

I **believe** that part of the problem here is the rewriter is not creating a representative compilation.  Instead it's essentially creating a one file project with no references.  This is likely causing the Simplifier to not be able to act here because it can't resolve the references.